docs/tools/update-git-versions.json.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+git_versions_file=$1
+
+if [[ -z $git_versions_file ]]
+then
+	echo "Usage: update-git-versions.json.sh <git-versions.json file> [--in-place]"
+	exit 1
+fi
+
+# Clone a fresh repository
+if [[ ! -d git ]]
```

Comment:
Maybe this directory should be explicitly ignored in `.gitignore` of the root repo to avoid confusion? Or clone it to `/tmp`?

Reply:
Sure, we could do that. Given what I said above (the script is just for maintainers), I didn’t put too much priority on polishing the script. Do you think I should improve this for this pull request?

Reply:
Nope, it's fine - it was just a nit 👍